### PR TITLE
modify the node qos and avoidance action type

### PR DESCRIPTION
**Pull request by yan234280533**

### CODE REVIEW

File: pkg/ensurance/analyzer/analyzer.go [hunk 1]
```diff
@@ -504,8 +503,14 @@ func combineEvictDuplicate(e *executor.EvictExecutor, evictPods executor.EvictPo
 		if i := e.EvictPods.Find(ep.PodKey); i == -1 {
 			e.EvictPods = append(e.EvictPods, ep)
 		} else {
-			if ep.DeletionGracePeriodSeconds < e.EvictPods[i].DeletionGracePeriodSeconds {
-				e.EvictPods[i].DeletionGracePeriodSeconds = ep.DeletionGracePeriodSeconds
+			if ep.DeletionGracePeriodSeconds != nil {
+				if e.EvictPods[i].DeletionGracePeriodSeconds == nil {
+					e.EvictPods[i].DeletionGracePeriodSeconds = ep.DeletionGracePeriodSeconds
+				} else {
+					if *(e.EvictPods[i].DeletionGracePeriodSeconds) > *(ep.DeletionGracePeriodSeconds) {
```
Comment (chenkaiyue) [inline pkg/ensurance/analyzer/analyzer.go:510]: maybe a little duplicated
```
if e.EvictPods[i].DeletionGracePeriodSeconds == nil || if *(e.EvictPods[i].DeletionGracePeriodSeconds) > *(ep.DeletionGracePeriodSeconds){
    e.EvictPods[i].DeletionGracePeriodSeconds = ep.DeletionGracePeriodSeconds
}
```

Comment (yan234280533) [reply pkg/ensurance/analyzer/analyzer.go:510]: if the ep.DeletionGracePeriodSeconds is nil, it will crash

Comment (yan234280533) [reply pkg/ensurance/analyzer/analyzer.go]: done~

**Pull request merged**